### ISSUE TITLE
consensus: Include ratification signature and bitset in block certificate

### ISSUE THREAD

**Issue by goshawk-3** [dusk-network/rusk#1027]
**Describe the bug**
As proved in #1020 , a single agreement message is completely enough for creating a block with valid certificate that is accepted by the network. 

We should consider adding `ratification` aggregate signature together with a bitset of voters  in the block certificate.

```rust
pub struct Certificate {
    pub first_reduction: StepVotes,
    pub second_reduction: StepVotes,
    pub ratification: StepVotes,  /// New aggregate signagure
}

pub struct StepVotes {
    pub bitset: u64,
    pub aggregate_signature: Signature,
}
```

**Additional context**
If this is issue is approved for implementation, we need first to address #1026 otherwise aggregate_signature and bitset from 2nd reduction can be reused.

**Comment by fed-franz** [comment]
This is an alternative solution to the ["accepting a block from a single Agreement"](https://github.com/dusk-network/dips/issues/2) proposal, right?
It should be deprecated if accepting the DIP

**Comment by herr-seppia** [comment]
Resolved in #1346
